_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp

```diff
@@ -1438,15 +1438,23 @@ class GetDefault
         , const mfxVideoParam& par
         , eMFXHWType hw)
     {
+        auto fcc = par.mfx.FrameInfo.FourCC;
```

Comment:
move it close to the place where it is used

Reply:
Thanks for the comment, I will update the PR.